e2e/Makefile

```diff
@@ -0,0 +1,11 @@
+DOCKER := $(shell which docker)
+TEST_CONTAINERS=$(shell docker ps --filter "label=ibc-test" -a -q)
+
+cleanup-ibc-test-containers:
+	for id in $(TEST_CONTAINERS) ; do \
+		$(DOCKER) stop $$id ; \
+		$(DOCKER) rm $$id ; \
+	done
+
+e2e-test: cleanup-ibc-test-containers
+	@go test -v ./ --run $(suite) -testify.m ^$(test)$$
```

Comment:
I'm not exactly a Makefile guru, but should these `.PHONY` targets as they're not creating an output file?

Reply:
you are absolutely correct, thanks for spotting that!